app/license.go

```diff
@@ -7,16 +7,29 @@ import (
 	"bytes"
 	"errors"
 	"net/http"
+	"os"
 	"strings"
 
 	"github.com/mattermost/mattermost-server/v5/mlog"
 	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/mattermost/mattermost-server/v5/utils"
 )
 
-const requestTrialURL = "https://customers.mattermost.com/api/v1/trials"
+const (
+	requestTrialURL = "https://customers.mattermost.com/api/v1/trials"
+	LicenseENV      = "MM_LICENSE"
```

Comment:
nit: should this be `LicenseEnv`? 0/5

Reply:
What happens with the current settings we have? for example, if the user sets the `ServiceSettings.LicenseFileLocation` and this new var?

side note: in the operator side, we will need to manage two ways to inject the license for versions `<= 5.25`

Reply:
The ENV var takes priority over LicenseFileLocation and any DB saved license. In other words, those are ignored as long as MM_LICENSE is defined with a non-empty string.